soh/include/z64actor.h

```diff
@@ -103,34 +103,34 @@ typedef struct {
     /* 0x18 */ Vec3f feetPos[2]; // Update by using `Actor_SetFeetPos` in PostLimbDraw
 } ActorShape; // size = 0x30
 
-#define ACTOR_FLAG_0 (1 << 0)
-#define ACTOR_FLAG_2 (1 << 2)
-#define ACTOR_FLAG_3 (1 << 3)
-#define ACTOR_FLAG_4 (1 << 4)
-#define ACTOR_FLAG_5 (1 << 5)
-#define ACTOR_FLAG_6 (1 << 6)
-#define ACTOR_FLAG_7 (1 << 7)
-#define ACTOR_FLAG_8 (1 << 8)
-#define ACTOR_FLAG_9 (1 << 9)
-#define ACTOR_FLAG_10 (1 << 10)
+#define ACTOR_FLAG_TARGETABLE (1 << 0) //ACTOR_FLAG_TARGETABLE
```

Comment:
Maybe it's obvious, but why is the flag name repeated in a comment after all of these?

Reply:
Remnant of setting flag names next to numeric values. Will remove once I'm done.